ext/hiredis_ext/extconf.rb

```diff
@@ -27,15 +33,21 @@
 end
 
 if build_hiredis
-  # Make sure hiredis is built...
-  Dir.chdir(hiredis_dir) do
-    success = system("#{make_program} static")
-    raise "Building hiredis failed" if !success
-  end
+  if config_system_libraries?
+    # Make sure that hiredis development modules are installed ...
+    find_header("hiredis.h", *["", "hiredis"].map {|p| File.join(RbConfig::CONFIG["includedir"], p) })
```

Comment:
Would we expect `hiredis.h` to be installed in the Ruby include dir? I'm wondering why we wouldn't just keep things simple and leave this as `find_header("hiredis.h")`.